### ISSUE TITLE
Test runs is lost and unable to retrieve it back

### ISSUE THREAD

**Issue by LintaEBaby** [SmartBear/cucumberstudio-issue-tracker#232]
### Steps to reproduce
1. Anyone is able to delete a test run directly once its actively on the execution cycle
2.
### Expected behaviour
If someone has deleted a test run. There should be options to retrieve the previous test run or support team should be able to do that. Only admin should be having the ability to delete a test run once its archived.

### Actual behaviour
Tell us what happens instead

### My configuration
**Browser:**
Google Chrome
**Operating system:**
Windows OS

**Comment by cbliard** [comment]
Hi Linta,

Many users had this problem initially, so we added a message before deletion to warn people about the deletion

![image](https://user-images.githubusercontent.com/176055/103866013-60569b00-50c5-11eb-8f89-3c13700087c4.png)

Then users were still mistakenly deleting test runs, so we added a "Archive" action instead of delete.

![image](https://user-images.githubusercontent.com/176055/103866099-83814a80-50c5-11eb-80d1-8f026565bfe2.png)

That works like a deletion: you archive your test run so that it does not get in the way anymore.

So you're suggesting to add a kind of undelete test run, meaning that once it's archive and deleted, it is still alive and visible somewhere ready to be restored if needed, right?

**Comment by LintaEBaby** [comment]
Yes, that's right. Now what happened was the test run got deleted and it was an active test run. It wasn't an archived one.
So now there is no option to undelete it. I also contacted the CucumberStudio support guys to retrieve the data back as it was many days worth work.
But they were also unable to do so. 
My suggestion is to enable such an option to undelete atleast for Admin users. So that by mistake if anyone has deleted an active test run. Admin can restore it anytime. Thanks
